data-types/collections/spec/arrays_spec.rb

```diff
@@ -3,174 +3,174 @@
   it 'test 0' do
     # in the line below, create a new empty array
 
-    empty = ______
+    empty = Array.new
     expect(empty).to eq([])
   end
 
-  xit 'test 1' do
+  it 'test 1' do
     # in the line below, create an array with the numbers 1 through 5
-    nums = _____
+    nums = Array.new(5) {|index| index + 1}
```

Comment:
Nice use of the curly boys. Generally, there is a space at the beginning and end of the curly boys for readability,.